t/clean-event.t

```diff
@@ -293,4 +293,37 @@ note( "mismatched and misnested tags" );
     $clean->();
     is ( $orig_post, $clean_post, "Full text of entry, with mismatched HTML tags within and with-out the cut" );
 }
+
+note("expected wordbreak behavior");
+
+$orig_post  = qq{wordbreak};
+$clean_post = qq{word<wbr />brea<wbr />k};
+$clean->({ wordlength => 4 });
+is( $orig_post, $clean_post, "Word break tags inserted where requested" );
+
+$orig_post  = qq{insert a word<wbr>break};
+$clean_post = qq{insert a word<wbr>break};
+$clean->();
+is( $orig_post, $clean_post, "Existing word break tags unchanged" );
+
+$orig_post  = qq{word-break};
+$clean_post = qq{word-<wbr />break};
+$clean->({ wordlength => 8 });
+is( $orig_post, $clean_post, "Word break tag prefers punctuation points" );
+
+$orig_post  = qq{&quot;entity&quot; test};
+$clean_post = qq{&quot;entity&quot; test};
+$clean->({ wordlength => 8 });
+is( $orig_post, $clean_post, "Word break handling of HTML entities OK" );
+
+$orig_post  = qq{multi-character-string test};
+$clean_post = qq{multi-character-<wbr />string test};
+$clean->({ wordlength => 20 });
+is( $orig_post, $clean_post, "Choose last punctuation in string" );
+
+$orig_post  = qq{"This_is_a_test_of_the_emergency_word_break_system."};
+$clean_post = qq{"This_is_a_test_of_the_emergency_word_br<wbr />eak_system."};
```

Comment:
I used the underscore character in this test because it's a special character included in \w, and I was making sure the word break wasn't inserted after the initial quotation mark.